stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -114,23 +138,74 @@ class PaymentSheet internal constructor(
         }
     }
 
+    /**
+     * A class that presents the individual steps of a payment sheet flow.
+     */
     internal interface FlowController {
-        fun getPaymentOption(): PaymentOption?
 
-        fun configure(
+        /**
+         * Configure the FlowController to process a [PaymentIntent].
+         *
+         * @param[paymentIntentClientSecret] the client secret for the [PaymentIntent].
+         * @param[configuration] optional [PaymentSheet] settings.
+         * @param[callback] called with the result of configuring the FlowController.
+         */
+        fun configureWithPaymentIntent(
             paymentIntentClientSecret: String,
-            configuration: Configuration,
+            configuration: Configuration? = null,
             callback: ConfigCallback
         )
 
-        fun configure(
+        /**
+         * @see [configureWithPaymentIntent] above.
+         */
+        fun configureWithPaymentIntent(
             paymentIntentClientSecret: String,
             callback: ConfigCallback
+        ) {
+            configureWithPaymentIntent(paymentIntentClientSecret, null, callback)
+        }
```

Comment:
For consistency can we avoid defining a default interface method implementation? Also have you verified if this works in Java?

Reply:
Isn't this what @JvmOverloads do? It just calls the method with the most parameters. https://medium.com/android-news/demystifying-the-jvmoverloads-in-kotlin-10dd098e6f72
I have not verified. I can check.

Reply:
Removed overloads.